src/terms/hubbard.jl

```diff
@@ -195,10 +214,11 @@ function compute_hubbard_n(manifold::ResolvedOrbitalManifold, projectors, labels
             hubbard_n[σ, idx, jdx] = mpi_sum(hubbard_n[σ, idx, jdx], basis.comm_kpts)
         end
     end
-    hubbard_n = symmetrize_hubbard_n(basis.model, manifold, hubbard_n; basis.symmetries)
+    symmetrize_hubbard_n(basis.model, manifold, hubbard_n; basis.symmetries)
 end
-function compute_hubbard_n(term::TermHubbard, basis::PlaneWaveBasis, ψ, occupation)
-    compute_hubbard_n(term.manifold, term.P, term.labels, basis, ψ, occupation)
+function compute_hubbard_ns(term::TermHubbard, basis::PlaneWaveBasis, ψ, occupation)
```

Comment:
We could keep the name for consistency with usage of `hubbard_n` elsewhere in the code.